src/js/components/JobSearchFilter.js

```diff
@@ -0,0 +1,65 @@
+import mixin from 'reactjs-mixin';
+/* eslint-disable no-unused-vars */
+import React from 'react';
+/* eslint-enable no-unused-vars */
+
+import FilterInputText from './FilterInputText';
+import QueryParamsMixin from '../mixins/QueryParamsMixin';
+import JobFilterTypes from '../constants/JobFilterTypes';
+
+const METHODS_TO_BIND = ['setSearchString'];
+
+class JobSearchFilter extends mixin(QueryParamsMixin) {
+  constructor() {
+    super();
```

Comment:
Please pass the arguments to the super class `super(...arguments);` as discussed in https://github.com/dcos/dcos-ui/pull/221#discussion_r63978069 and https://github.com/dcos/dcos-ui/pull/221#discussion_r64065922